test/Main

```diff
@@ -0,0 +1,6 @@
+#include "UnitTest++/UnitTestPP.h"
```

Comment:
Tests should be saved for later; lets prioritize the view

Reply:
gotcha i'll remove them